app/views/measures/grouped/_table.html.erb

```diff
@@ -1,4 +1,4 @@
-<h3 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-3", id=<%= local_assigns[:css_id]%>>
+<h3 class="govuk-heading-m govuk-!-margin-top-5 govuk-!-margin-bottom-3", id=<%= local_assigns[:css_id]%>>
```

Comment:
I'm guessing this margin top change is to match Matt's prototype?

Reply:
Yes, otherwise, the h3 titles would be far from the following table.
Without it, it looks bad.

Reply:
:tada: